datasets/_global/afdb_sanctions/crawler.py

```diff
@@ -1,7 +1,45 @@
+import re
+
 from zavod import Context
 from zavod import helpers as h
 from zavod.shed.zyte_api import fetch_html
 
+NAME_SPLITS = [
+    "doing business as",
+    "also known as",
+    " or ",
+    "a.k.a.",
+    "aka ",
+    "f/k/a",
+    "d/b/a",
+    "dba",
+    "d.b.a.",
+    "a/k/a",
+    "operating",
+    "formerly",
+    "previously",
+]
+RE_ALIAS = re.compile("|".join(NAME_SPLITS), re.IGNORECASE)
+NAME_REGEX = re.compile(r"^(?P<name>[^()]+?)\s*\(\s*(?P<alias>[^()]+?)\s*\)$")
+
+
+def apply_clean_name(context: Context, entity, name):
+    if "(" in name and NAME_REGEX.match(name) and not RE_ALIAS.search(name):
```

Comment:
(nit) I have a suggestion for how to restructure this cascading `if`, but use what you prefer

https://github.com/opensanctions/opensanctions/commit/1f538a297e0dd8ad63ca553ffa86b88c592803e6